config/v1/tests/ingresses.config.openshift.io/SetEIPForNLBIngressController.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "Ingress"
+crdName: ingresses.config.openshift.io
+tests:
```

Comment:
[The integration job failed](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_api/1826/pull-ci-openshift-api-master-integration/1783296584281231360#1:build-log.txt%3A8015), and it looks like the reason is that it is running these tests for the "Default" featureset.  Try adding the following to this file:
```yaml
featureGate: SetEIPForNLBIngressController
```